WalletWasabi.Fluent/Views/AddWallet/Create/RecoveryWordsView.axaml

```diff
@@ -15,26 +15,44 @@
                  EnableBack="{Binding EnableBack}"
                  EnableNext="True" NextContent="Continue" FocusNext="True"
                  ScrollViewer.HorizontalScrollBarVisibility="Auto">
-    <ItemsControl Items="{Binding MnemonicWords}" VerticalAlignment="Center" HorizontalAlignment="Center">
-      <ItemsControl.Styles>
-        <Style Selector="TextBlock">
-          <Setter Property="FontSize" Value="{DynamicResource FontSizeH6}" />
-          <Setter Property="Height" Value="42" />
-        </Style>
-      </ItemsControl.Styles>
-      <ItemsControl.ItemsPanel>
-        <ItemsPanelTemplate>
-          <WrapPanel MaxHeight="168" Orientation="Vertical" VerticalAlignment="Top" HorizontalAlignment="Center" />
-        </ItemsPanelTemplate>
-      </ItemsControl.ItemsPanel>
-      <ItemsControl.ItemTemplate>
-        <DataTemplate>
-          <StackPanel Orientation="Horizontal" Margin="20 0">
-            <TextBlock Text="{Binding Index, Mode=OneWay, StringFormat={}{0}.}" />
-            <TextBlock Text="{Binding Word}" FontWeight="SemiBold" Margin="5 0 0 0" />
-          </StackPanel>
-        </DataTemplate>
-      </ItemsControl.ItemTemplate>
-    </ItemsControl>
+    <Panel>
+      <ItemsControl Items="{Binding MnemonicWords}" VerticalAlignment="Center" HorizontalAlignment="Center" x:Name="ItemsControl">
+        <ItemsControl.Styles>
+          <Style Selector="TextBlock">
+            <Setter Property="FontSize" Value="{DynamicResource FontSizeH6}" />
+            <Setter Property="Height" Value="42" />
+          </Style>
+        </ItemsControl.Styles>
+        <ItemsControl.ItemsPanel>
+          <ItemsPanelTemplate>
+            <WrapPanel MaxHeight="252" Orientation="Vertical" VerticalAlignment="Top" HorizontalAlignment="Center" />
+          </ItemsPanelTemplate>
+        </ItemsControl.ItemsPanel>
+        <ItemsControl.ItemTemplate>
+          <DataTemplate>
+            <StackPanel Orientation="Horizontal" Margin="50 0 50 0" Width="80">
+              <TextBlock Text="{Binding Index, Mode=OneWay, StringFormat={}{0}.}" Opacity="0.5" />
+              <TextBlock Text="{Binding Word}" FontWeight="SemiBold" Margin="5 0 0 0" />
+            </StackPanel>
+          </DataTemplate>
+        </ItemsControl.ItemTemplate>
+      </ItemsControl>
+
+      <Panel HorizontalAlignment="Center" VerticalAlignment="Center"
+             Width="{Binding #ItemsControl.Bounds.Width}"
+             Height="{Binding #ItemsControl.Bounds.Height}">
+        <Separator Classes="vertical" Margin="0 0 0 20" />
+      </Panel>
+
+      <c:InfoMessage Foreground="{StaticResource WarningMessageForeground}"
+                         Margin="0,16,0,0"
+                         HorizontalAlignment="Center"
+                         VerticalAlignment="Bottom">
+        <StackPanel>
+          <TextBlock Text="Please make sure to safely store these Recovery Words." />
```

Comment:
```suggestion
          <TextBlock Text="Make sure to safely store these Recovery Words." />
```

Are we so polite and say "please" elsewhere too?

Reply:
this comment inspired me to write https://github.com/zkSNACKs/WalletWasabi/issues/9084 lol